.hooks/pre-push

```diff
@@ -17,7 +17,7 @@ npx eslint --quiet
 printf "\n⏳ npm run test\n"
 npm run test
 
-printf "\n⏳ composer test\n"
-vendor/bin/phpunit
+printf "\n⏳ composer paratest\n"
+vendor/bin/paratest
```

Comment:
is there a difference between this and `composer paratest`, which runs `php artisan test --parallel`?

Reply:
by running it through composer it might be subjetct to composer's execution time limit. realistically we could switch all of those commands to go through composer scripts as the echo statements imply.